contrib/cmake/src/test/lp/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+add_executable(lp_tst lp_main.cpp lp.cpp $<TARGET_OBJECTS:util> $<TARGET_OBJECTS:polynomial> $<TARGET_OBJECTS:nlsat>  $<TARGET_OBJECTS:lp> )
```

Comment:
This file is in the wrong location. I suggest you rebase on to the latest master.